packages/odf/components/create-storage-system/create-storage-system-steps/security-and-network-step/security-and-network-step.tsx

```diff
@@ -6,13 +6,17 @@ import { Form } from '@patternfly/react-core';
 import { FEATURES } from '../../../../features';
 import { NetworkType, NADSelectorType } from '../../../../types';
 import { WizardDispatch, WizardState } from '../../reducer';
+import { ConnectionDetails } from '../connection-details-step';
 import { NetworkFormGroup } from './configure';
 import { Encryption } from './encryption';
 
 export const SecurityAndNetwork: React.FC<SecurityAndNetworkProps> = ({
-  state,
+  securityAndNetworkState,
   dispatch,
   infraType,
+  isExternal,
+  connectionDetails,
```

Comment:
nits: just to be consistent with "securityAndNetworkState" naming...
```suggestion
  connectionDetailsState,
```